components/crud-web-apps/volumes/backend/apps/default/routes/get.py

```diff
@@ -0,0 +1,15 @@
+from kubeflow.kubeflow.crud_backend import api, logging
+
+from ...common import utils
+from . import bp
+
+log = logging.getLogger(__name__)
+
+
+@bp.route("/api/namespaces/<namespace>/pvcs")
+def get_pvcs(namespace):
+    # Return the list of PVCs and the corresponding Viewer's state
```

Comment:
How are the Viewer's state handled here?

Reply:
Good catch, I'll need to format this misleading comment. The viewer functionality is not present in the default flavor in this iteration so I'll need to rephrase here

Reply:
Pushed a commit to remove the misleading part of the comment